Project.toml

```diff
@@ -34,11 +33,27 @@ Zeros = "bd1ec220-6eb4-527a-9b49-e79c3db6233b"
 ZygoteRules = "700de1a5-db45-46bc-99cf-38207098b444"
 
 [weakdeps]
+<<<<<<< HEAD
+<<<<<<< HEAD
+HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
+VectorInterface = "409d34a3-91d5-4945-b6ec-7529ddf182d8"
```

Comment:
Seems like the merge got messed up and `Observers` got dropped.

Reply:
Oof, that is bad. I will fix.